### DISCUSSION TITLE
Monitoring partition creation in RDS

### DISCUSSION THREAD

**Discussion by vbhosle** [pgpartman/pg_partman#612] [General]
Hi. We use Postgres RDS. It doesn't support pg_jobmon. So we are looking for alternatives to generate alerts when partition creation fails or not created for some reason. 
One thing that comes to my mind is setting alert on default partition when inserts happen on it. 
Is anyone already in same situation and managed to put reliable monitoring on partition creation?

**Comment by keithf4** [comment]
If you hadn't seen it, you can use the `check_default()` function to provide the monitoring on the default partition.
Another method, if you have retention enabled, would be to count the number of child tables in any given set and ensure it is always above a minimum number.

**Reply by keithf4** [reply]
You can use the `show_partitions()` function to help provide the child table count a little easier.

**Reply by vbhosle** [reply]
Thanks! This helps. I will get back if there are any gaps after using it for monitoring.
